protokube/pkg/protokube/volume_mounter.go

```diff
@@ -52,6 +52,12 @@ func (k *VolumeMountController) mountMasterVolumes() ([]*Volume, error) {
 	}
 
 	for _, v := range attached {
+		// Do not ever try to mount root devices
+		if v.LocalDevice == "/dev/sda1" || v.LocalDevice == "/dev/xvda" {
```

Comment:
I think in this case an infinite loop is a good outcome - we're trying to format the root partition.

If you really want this here, how about:
```
// Do not ever try to mount root devices
if v.LocalDevice == "/dev/sda1" || v.LocalDevice == "/dev/xvda" {
  glog.Fatalf("Attempted to mount root device")
}
```

But IMO, it's just not worth it ... we're as likely one day to legitimately mount on `/dev/xvda`.